package.json

```diff
@@ -15,30 +15,46 @@
   "bin": {
     "avocado": "./bin/cli.js"
   },
+  "standard": {
+    "parser": "typescript-eslint-parser",
+    "plugins": [
+      "typescript"
+    ]
+  },
   "scripts": {
     "integration": "tsc && cd ../azure-rest-api-specs/ && node ../avocado/bin/cli.js",
     "cli": "tsc && node ./bin/cli.js",
     "tsc": "tsc",
     "prettier": "prettier ./src/**/*.ts",
     "tslint": "tslint --project ./",
-    "tslint-check": "tslint-config-prettier-check ./tslint.json",
-    "test": "tsc && tslint --project ./ && nyc mocha ./dist/test/*.js --reporter mocha-multi-reporters --reporter-options configFile=mocha-multi-reporters.json",
+    "test": "tsc && tslint --project ./ && jest",
+    "standard": "standard src/**/*.ts",
     "prepack": "npm install && tsc"
   },
-  "nyc": {
-    "reporter": [
-      "html",
-      "text",
-      "cobertura"
+  "jest": {
+    "testMatch": [
+      "**/dist/test/*.js"
     ],
-    "include": [
-      "dist/*.js"
+    "reporters": [
+      "jest-junit",
+      "default"
     ],
-    "check-coverage": true,
-    "lines": 100,
-    "statements": 100,
-    "functions": 100,
-    "branches": 100
+    "collectCoverage": true,
+    "coverageThreshold": {
```

Comment:
can you add "testEnvironment": "node" so it doesnt use jsdom env and node instead ( it will be faster )